Resources/Locale/en-US/_NF/bluespace-events/events.ftl

```diff
@@ -1,7 +1,7 @@
-station-event-bluespace-vault-start-announcement = A NanoTrasen autonomous armored vault transport has made an FTL error and will soon arrive nearby. Your sector will be rewarded for it's safe return.
+station-event-bluespace-vault-start-announcement = A NanoTrasen autonomous armored vault transport has made an FTL error and will soon arrive nearby. Your sector will be rewarded for it's safe return. A No Fly Zone of 300 meters has been established. Failure to comply will result in prosecution
 station-event-bluespace-vault-end-announcement = We have successfully retrieved the vault from your sector and have reimbursed your nearby Frontier Outpost accordingly.
```

Comment:
>Wouldn't "A No Fly Zone may be declared." do the job?

It would, but that falls down to the NFSD to perform, and at least in my experiences it leads to frequent arguing about it.
I'm fine with it either way though.

Reply:
The range is too much, my line of thinking was by having the NFZ’s announced by the game announcement rather than by a player would stop any arguments about people getting angry at an SR or sherif for declaring an NFZ, or angry at them for not declaring an NFZ, in the case of other NFSD members.

Reply:
@Leander-0 had made an idea thread on the discord regarding it, can ping you later.